cove_360/templates/cove_360/explore.html

```diff
@@ -173,17 +173,27 @@ <h3 class="panel-title panel-title-explore">
         <span class="glyphicon glyphicon-collapse-down pull-right"></span>
       </h3>
     </div>
-    <div id="additional-checks-body" class="collapse">
+    <div id="additional-checks-body" class="collapse panel-body">
         {% if additional_checks_errored %}
         <div class="row"> <!--Start Row (Detail Table)-->
           <div class="col-xs-12">
             <div class="alert alert-danger">
-              {%blocktrans%}Additional checks failed due to an issue with the data. Fix some validation errors first.{%endblocktrans%}
+              {% blocktrans %}Additional checks failed due to an issue with the data. Fix some validation errors first.{% endblocktrans %}
             </div>
           </div>
         </div>
         {% endif %}
         {% if additional_checks %}
+        <p class="explanation">{% blocktrans %}Additional checks have been made on your data.{% endblocktrans %}</p>
+        <br>
+        <p>
+            {% blocktrans %}
```

Comment:
Has this copy been signed off by 360?

Reply:
It is a minor variation of this:
https://github.com/OpenDataServices/cove/issues/672#issuecomment-299483191
by @KDuerden. 

It reads:

>Additional checks have been made on your data.
Additional checks highlight areas where the source data may be incorrect or require further attention. They can also suggest ways to increase the usefulness of the data. In any case, these checks do not indicate validation errors and can be ignored if they are not relevant.

@KDuerden, is that OK?

Reply:
@KDuerden I will add _"some"_ to that line. Any other final tweak before we consider this good to go?

Reply:
Great - good to go. Thank you.